### PR TITLE
25 sign up screen UI

### PR DESCRIPTION
**Pull request by Njumbi**

### CODE REVIEW

File: app/src/main/java/com/todoist_android/ui/auth/AuthenticationViewModel.kt [hunk 1]
```diff
@@ -20,20 +22,13 @@ class AuthenticationViewModel(
      * and then we will use the data to update the UI
      ***/
 
-    private val _loginResponse : MutableLiveData<APIResource<LoginResponse>> = MutableLiveData()
-    private val _signUpResponse : MutableLiveData<APIResource<SignupResponse>> = MutableLiveData()
+    // use kotlin flows instead of live data
+    private val _loginResponse = MutableSharedFlow<APIResource<LoginResponse>>()
```
Comment (malcolmmaima) [inline app/src/main/java/com/todoist_android/ui/auth/AuthenticationViewModel.kt:27]: @Njumbi what of signupResponse?

File: app/src/main/java/com/todoist_android/ui/auth/LoginFragment.kt [hunk 2]
```diff
@@ -42,18 +64,5 @@ class LoginFragment : BaseFragment<AuthenticationViewModel, FragmentLoginBinding
             //TODO: add validation
             viewModel.login(email, password)
         }
-
-        binding.textViewRegister.setOnClickListener {
```
Comment (malcolmmaima) [inline app/src/main/java/com/todoist_android/ui/auth/LoginFragment.kt:46]: navigating to Signup Fragment with clickable link ... we won't need this?

**Review by malcolmmaima — APPROVED** [review]
Some minor comments but looks good to me

File: app/src/main/java/com/todoist_android/ui/auth/AuthenticationViewModel.kt (hunk 1 above)
Comment (Njumbi) [reply app/src/main/java/com/todoist_android/ui/auth/AuthenticationViewModel.kt:27]: I haven't implemented sign up ,  I don't think I have the recent changes.

File: app/src/main/java/com/todoist_android/ui/auth/LoginFragment.kt (hunk 2 above)
Comment (Njumbi) [reply app/src/main/java/com/todoist_android/ui/auth/LoginFragment.kt:46]: Let me pull to get the recent changes

**Comment by malcolmmaima** [comment]
[Sign Up Screen -Android](https://trello.com/c/z41mVux0/25-sign-up-screen-android)

**Pull request merged**